src/eradiate/data/_safe_online.py

```diff
@@ -33,6 +33,10 @@ class SafeOnlineDataStore(DataStore):
     registry_fname : path-like, optional
         Path to the registry file, relative to `path`.
 
+    retry_if_failed : int, default: 1
```

Comment:
We probably want the default value to be 3. Users can't care less about our API at that point, they would not even notice the retry feature if we keep using a retry count of 1. Advanced users who would want to optimize their IOs over multiple distributed hosts will probably automate this task of downloading the data and would like to make it run only once, trying to avoid script failures as much as possible too.